source/regex.tex

```diff
@@ -2427,7 +2382,8 @@
 
 \begin{itemdescr}
 \pnum
-\requires \tcode{ready() == true} and \tcode{OutputIter} shall meet the requirements for a
+\expects
+\tcode{ready() == true} and \tcode{OutputIter} meets the requirements for a
```

Comment:
```suggestion
\tcode{ready() == true} and \tcode{OutputIter} meets the requirements for an
```

Reply:
`a` is right since `\oldconcept` adds the _Cpp17_ prefix.